homeassistant/components/soma/__init__.py

```diff
@@ -54,7 +54,9 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Soma from a config entry."""
     hass.data[DOMAIN] = {}
-    hass.data[DOMAIN][API] = SomaApi(entry.data[HOST], entry.data[PORT])
+    hass.data[DOMAIN][API] = await hass.async_add_executor_job(
+        SomaApi, entry.data[HOST], entry.data[PORT]
+    )
     devices = await hass.async_add_executor_job(hass.data[DOMAIN][API].list_devices)
     hass.data[DOMAIN][DEVICES] = devices["shades"]
```

Comment:
I think this would be prettier:
```python
    api = await hass.async_add_executor_job(SomaApi, entry.data[HOST], entry.data[PORT])
    devices = await hass.async_add_executor_job(api.list_devices)
    hass.data[DOMAIN] = {API: api, DEVICES: devices["shades"]}
```

Reply:
I added the requested changes to code (sorry, not a Python dev most of the time so I'm really not sure what I'm doing most of the time).

Unfortunately I don't really have a link to to release notes at this time. The pysoma library is only really used by this integration as far as I know and I don't have the repo for that publicly available anywhere. The whole library is really just a wrapper around some HTTP calls. If it is really necessary I can probably set up some sort of release notes document somewhere but in this case it would be just a formality.

Reply:
> For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description

Reply:
I have added the link (https://support.somasmarthome.com/hc/en-us/categories/360001067893-SOMA-Connect) for the changelog to the first section now. Is that the right place for it?